tests/test_backend_features.py

```
@@ -0,0 +1,166 @@
+
+import requests
+
+
+def _p(root, *parts):
+    return os.path.join(root, *parts)
+
+
+# ===== #5 打包下载 =====
+
+def test_download_zip_small_memory(live_server):
+    """小目录内存打包：漫画A 下 5 张图能被 zipfile 正常打开，namelist 含 5 张。"""
```

Ruff (0.15.20)

[warning] Docstring contains ambiguous `：` (FULLWIDTH COLON). Did you mean `:` (COLON)?

(RUF002)